corpus/accounts/models.py

```diff
@@ -140,7 +140,7 @@ class ExecutiveMember(models.Model):
     )
     is_nep = models.BooleanField(default=False, verbose_name="Is NEP Member?")
     date_joined = models.DateTimeField(
-        default=timezone.now(), verbose_name="Date Joined"
+        default=timezone.localtime(), verbose_name="Date Joined"
```

Comment:
```suggestion
        default=timezone.localtime, verbose_name="Date Joined"
```

This shouldn't be a function call, because the function call returns the actual `datetime` object each time a migration is run.